src/issue.js

```diff
@@ -1,71 +1,75 @@
 console.time('codesy issue load');
 
-const make_widget = ({domain}) => {
-    return new Promise((resolve)=>{
-        const codesy = {
-            css:
-                {attr: {
-                    rel: "stylesheet",
-                    type: "text/css"
-                }
-            },
-            iframe:
-                { attr:{
-                    id: "codesy_iframe",
-                    style: "visibility: collapse;",
-                    scrolling: "no",
-                    seamless: "seamless"
-                }
+const make_widget_appender = (domain) => {
```

Comment:
artistic license?  Actually, I'm settling on named functions if it longer than one line.